structured_experiments/2022.04.28.notebook_template/experiment.MD

```diff
@@ -0,0 +1,41 @@
+<!--DESC Template folder to use for experiments-->
+# Hypothesis
+TODO: Create Template
+
+# Methodology
+
+TODO: Create Template
+
+# Results
+
+TODO: Create Template
+
+# Conclusions
+
+TODO: Create Template
+
+
+
+# Reproduce
+
+## 1. First Acquire the training data
+
+### 
+Run `git lfs pull --exclude .git -I  data/storysquad-ground-truth/`
+
+This will use git-lfs to pull down all the client given training images
+
+## 2. Execute the notebook container
+

```

Comment:
If a duplicate is created, the container name need to be change (Working locally). Need to add this information here